src/Apps/Artwork/Components/ArtworkPageBanner.tsx

```diff
@@ -87,9 +96,76 @@ const UnpublishedArtworkBanner = () => (
   </FullBleedBanner>
 )
 
+interface OrderBannerProps {
+  order: {
+    readonly internalID: string
+    readonly buyerState: string | null | undefined
+    readonly displayTexts: {
+      readonly title: string
+    }
+  }
+}
+
+const OrderBanner: FC<React.PropsWithChildren<OrderBannerProps>> = ({
+  order,
+}) => {
+  let variant: BannerVariant = "defaultLight"
+  let copy = "You ordered this artwork."
+  let linkCopy = "View order"
+
+  switch (order.buyerState) {
+    case "APPROVED":
+      copy = "Your order has been approved!"
+      variant = "success"
+      break
+    case "COMPLETED":
+      copy = "Your order is complete!"
+      break
+    case "OFFER_RECEIVED":
+      copy = "The gallery has responded to your offer."
+      linkCopy = "View offer"
+      variant = "defaultDark"
+      break
+    case "PAYMENT_FAILED":
+      copy = "Your payment failed."
+      linkCopy = "Update payment information"
+      variant = "error"
+      break
+    case "PROCESSING_OFFLINE_PAYMENT":
+      copy = "Your order is being processed."
+      linkCopy = "Please complete the payment."
+      variant = "brand"
+      break
+    case "REFUNDED":
+      copy = "Your order has been refunded."
+      break
+    case "SHIPPED":
+      copy = "Your order has been shipped!"
+      break
+    case "PROCESSING_PAYMENT":
+      copy = "Your payment is being processed."
+      break
+  }
+
+  return (
+    <FullBleedBanner variant={variant}>
```

Comment:
This is the component that needs the query renderer (or better, a hook -- useClientQuery or useLazyQueryRenderer)